### DISCUSSION TITLE
New releases are missing on Releases page

### DISCUSSION THREAD

**Discussion by ris58h** [community/community#59533] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

[My project](https://github.com/ris58h/WebCalm) has 4 releases (at this moment) and `0.8` is the latest release as you can see on the picture below:
![Screenshot 2023-06-28 at 12 05 22](https://github.com/community/community/assets/511242/c9277e99-5810-40ee-b413-464843f0e96f)

But on [Releases](https://github.com/ris58h/WebCalm/releases) page only 3 releases are available:
<img width="588" alt="Screenshot 2023-06-28 at 12 08 13" src="https://github.com/community/community/assets/511242/61086a95-9ff3-4e61-9bb8-346465feca67">

I faced the issue while I was preparing a draft release - it was missing on Releases page but was available on a direct link. After publishing the release is still missing.

**Comment by aonez** [comment]
Same as #59531

**Reply by piotrooo** [reply]
And #59535

**Reply by ris58h** [reply]
You have to wait https://github.com/orgs/community/discussions/59535#discussioncomment-6302989
